packages/client/tests/functional/order-by-null/tests.ts

```diff
@@ -36,121 +36,137 @@ testMatrix.setupTestSuite(
       })
     })
 
-    test('should return records sorted by name asc and null first', async () => {
-      const records = await prisma.user.findMany({
-        select: {
-          name: true,
-        },
-        orderBy: {
-          name: {
-            sort: 'asc',
-            nulls: 'first',
+    // TODO: Fails with TypeError: undefined cannot be passed as argument to the database
```

Comment:
This is the same error spotted above. With `wasm`, for some reason, we're emitting `undefined` args rather than `null` args to express `NULL` in SQL queries.